jpype/_core.py

```diff
@@ -339,6 +339,7 @@ def initializeResources():
     _jpype._type_classes[object] = _jpype._java_lang_Object
     _jpype._type_classes[_jpype.JString] = _jpype._java_lang_String
     _jpype._type_classes[_jpype.JObject] = _jpype._java_lang_Object
+    _jpype._type_classes[_jpype.JClass] = _jpype._java_lang_Class
```

Comment:
This seems reasonable.  Though technically all uses of `_type_classes` needs checking.  There used to be an edge case though I don't see it any more.   The only user is ``_jarray`` which i checked.